_data/icons/zksync-era.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://Qma6H9xd8Ydah1bAFnmDuau1jeMh5NjGEL8tpdnjLbJ7m2",
+    "width": 512,
+    "height": 512,
```

Comment:
identify Qma6H9xd8Ydah1bAFnmDuau1jeMh5NjGEL8tpdnjLbJ7m2         16:25:34
Qma6H9xd8Ydah1bAFnmDuau1jeMh5NjGEL8tpdnjLbJ7m2 SVG 517x517 517x517+0+0 8-bit sRGB 2308B 0.000u 0:00.009

Reply:
This is quite weird.. accessing it I get the following
<img width="612" alt="image" src="https://user-images.githubusercontent.com/8230135/219422817-60590cf9-b510-447c-ab4d-05d1056ebc0e.png">

Reply:
hm - weird - can you upload to iconDownload folder? In the end it needs to pass CI

Reply:
added

Reply:
How do I use this to update MetaMask ......   create a repository first ????